### DISCUSSION TITLE
How to open the program when my pc restart?

### DISCUSSION THREAD

**Discussion by Lewis1210-sudo** [lllyasviel/Fooocus#3543] [General]
my pc restarted, but i already installed in fooocus colab, now i don't know how to open the program again. should i install it again?

**Comment by IPv6** [comment]
If you have pay tier - you can connect to active colab instance again (if any, google for "Connect to a runtime")
For free tier you need to start colab from scratch again, it completely deleted after 1-2 hours by google anyway

**Reply by Lewis1210-sudo** [reply]
ohhh, i see, thanks
